Changelog.md

```diff
@@ -1,6 +1,8 @@
 ### 0.4.8 (unreleased)
 
 BugFixes:
+ * Remappings: a=b would overwrite c:a=d. This has now been fixed to all modules except
```

Comment:
This sounds quite complicated. What about `Prefer longer context over longer prefix.`